cypress/integration/accordions.spec.js

```diff
@@ -0,0 +1,100 @@
+const constants = {
+    rootUrl: 'https://www.bcpl.info/about-us/policies.html',
+    anchorUrl:
+        'https://bcpl.info/about-us/policies.html#accessibility-statement',
+    attrs: {
+        expanded: 'aria-expanded'
+    },
+    cssClasses: {
+        accordionCollapsedArrow: 'fa-chevron-right',
+        accordionExpandedArrow: 'fa-chevron-down',
+    },
+    selectors: {
+        accordionContainer: '.content-accordion',
+        accordionExpanded: '[aria-expanded="true"]',
+        accordionPanel: '.panel',
+        accessabilitySection: '#WebsiteGeneralUserTermsandConditions-panel-4'
+    }
+};
+
+describe('Accordion Component', () => {
+    const getFirstAccordionToggleBtn = () =>
+        cy
+            .get('@firstAccordion')
+            .find('a')
+            .first();
+
+    const toggleFirstAccordion = () =>
+        getFirstAccordionToggleBtn()
+            .click()
+            .wait(500);
+
+    before(() => {
+        cy.visit(constants.rootUrl);
+    });
+
+    beforeEach(() => {
+        cy.get(constants.selectors.accordionContainer).as('accordions');
+        cy
+            .get('@accordions')
+            .find(constants.selectors.accordionPanel)
+            .first()
+            .as('firstAccordion');
+    });
+
+    it('should at least one accordion on the page', () => {
+        cy.get('@firstAccordion').should('exist');
+    });
+
+    it('should expand when the accordion header is selected', () => {
+        toggleFirstAccordion()
+            .closest(constants.selectors.accordionPanel)
+            .find(constants.selectors.accordionExpanded)
+            .should('exist');
+    });
+
+    it('should show a down arrow icon when the accordion is expanded', () => {
+        getFirstAccordionToggleBtn()
+            .find('i')
+            .should('have.class', constants.cssClasses.accordionExpandedArrow);
+    });
+
+    it("should collapse an expanded accordion has it's header selected", () => {
+        toggleFirstAccordion()
+            .closest(constants.selectors.accordionPanel)
+            .find(constants.selectors.accordionExpanded)
+            .should('not.exist');
+    });
+
+    it('should show a down arrow icon when the accordion is expaned', () => {
```

Comment:
You have this test twice. Did you mean to change this to check for a right arrow when the accordion is collapsed?